engine/access/rest/common_query_params_test.go

```diff
@@ -0,0 +1,110 @@
+package rest
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/require"
+)
+
+// TestCommonQueryParamMiddlewares test the two common query param middlewares - expand and select
+func TestCommonQueryParamMiddlewares(t *testing.T) {
```

Comment:
test for the new middleware. Testing handles as described [here](https://github.com/gorilla/mux#testing-handlers)